unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/SettingsControllers/SpecificControllers/MaxNonLODAvatarsControlController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using UnityEngine;
+
+namespace DCL.SettingsControls
+{
+    [CreateAssetMenu(menuName = "Settings/Controllers/Controls/Max Non-LOD Avatars", fileName = "MaxNonLODAvatarsControlController")]
+    public class MaxNonLODAvatarsControlController : SliderSettingsControlController
+    {
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            KernelConfig.i.EnsureConfigInitialized()
```

Comment:
@pbosio we should do something about this approach.

1) KernelConfig is a singleton, and we should get rid of them as possible
2) Using it like this forces us to deal with race conditions
3) KernelConfig dirty state will be carried over from test to test, and that will force us to clear kernel config state on `IntegrationTestSuite`. At long term, this will make `IntegrationTestSuite` roll back to the `IntegrationTestSuite_Legacy` intialization clusterfuck.

Reply:
we can always move it to Environment like everything else